app/uk/gov/hmrc/economiccrimelevyregistration/views/templates/Layout.scala.html

```diff
@@ -96,7 +99,7 @@
 
 
 @additionalScripts = {
-    <script type="module" src='@controllers.routes.Assets.versioned("javascripts/application.min.js")'></script>
```

Comment:
This was recommended. Why remove?

Reply:
Sorry about that, removed it accidentally, rollbacked now